### PR TITLE
Solves bug in limit

### LINKED ISSUE #18482: Incorrect evaluation of limit
`limit(sqrt (x**2 + 6*x) + (x**3 + x**2)/(x**2 + 1),x,-oo)` is `2`
`limit((x**3 + x**2 + sqrt(x*(x + 6))*(x**2 + 1))/(x**2 + 1),x,-oo)` is `1`, but should be `-2`, since expressions are equivalent.

### PR DESCRIPTION
**Pull request by Saanidhyavats**
#### References to other Issues or PRs
 Fixes #18482

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Saanidhyavats** [comment]
After going through the code I found that removing certain lines resulted the expected value. On removing these codes and after running the tests everything seems to be fine (some 4 tests were failing in both the cases that is before and after updating the code)

File: sympy/series/tests/test_gruntz.py [hunk 1]
```diff
@@ -467,3 +467,7 @@ def test_issue_6682():
 def test_issue_7096():
     from sympy.functions import sign
     assert gruntz(x**-pi, x, 0, dir='-') == oo*sign((-1)**(-pi))
+
+def test_issue_18482():
+    _p = Symbol('_p', positive=True)
```
Comment (Smit-create) [inline sympy/series/tests/test_gruntz.py]: This needs to be removed

Comment (Saanidhyavats) [reply sympy/series/tests/test_gruntz.py]: Shall I use x instead of _p?

Comment (Smit-create) [reply sympy/series/tests/test_gruntz.py]: You have defined  `_p` but used x. Anyway, use `x` and remove this line

**Comment by Smit-create** [comment]
Also, please add the tests from [here](https://github.com/sympy/sympy/issues/18482#issue-555554662) to validate the change you have made.

File: sympy/series/tests/test_gruntz.py [hunk 2]
```diff
@@ -467,3 +467,6 @@ def test_issue_6682():
 def test_issue_7096():
     from sympy.functions import sign
     assert gruntz(x**-pi, x, 0, dir='-') == oo*sign((-1)**(-pi))
+
+def test_issue_18482():
+    assert gruntz((2*exp(3*x)/(exp(2*x) + 1))**(1/x),x,oo) == E
```
Comment (Smit-create) [inline sympy/series/tests/test_gruntz.py]: Add spaces after commas, see PEP8 style.

**Comment by Saanidhyavats** [comment]
@Smit-create I have worked on the single problem  `limit((2*exp(3*x)/(exp(2*x) + 1))**(1/x),x,oo)`
, will start working on other problems as well

**Comment by Sc0rpi0n101** [comment]
I don't think this fix needs to be updated in the release notes. None of the changes seem to affect the end-users.

**Comment by czgdp1807** [comment]
If it were to be added to release notes then something more descriptive should be added. Though no APIs are introduced or removed, release notes may be left empty.

**Comment by Saanidhyavats** [comment]
@czgdp1807 but its showing error  in `sympy-bot/release-notes` on leaving release notes empty

**Comment by mohitacecode** [comment]
You have to write "NO ENTRY" there.

**Comment by oscarbenjamin** [comment]
The test failure suggests that the code that is removed here is needed

**Comment by Saanidhyavats** [comment]
The code which was removed was able to get the correct results but simultaneously it lead to error in some other tests. I have started working on this and will make the changes.

**Comment by czgdp1807** [comment]
@Saanidhyavats Any news?

**Comment by Saanidhyavats** [comment]
@czgdp1807 I am currently going through the gruntz thesis to solve this and other issues. Will start working on this after completing the thesis within 1-2 days.

**Pull request closed without merging**